test/scripts/README.md

```diff
@@ -0,0 +1,75 @@
+# Bristlemouth HIL Test Scripts
+
+The following directory contains hardware-in-the-loop (HIL) test scripts that utilize [pytest](https://docs.pytest.org/en/stable/).
+This includes specific tests,
+as well as helper files to run the tests.
+
+## Setup
+
+In order to run HIL testing,
+[conda](https://docs.conda.io/en/latest/) is used to manage dependencies.
+The following will create a conda environment and download/install all the dependencies:
+
+```
+$ conda create -n bm_hil
+$ conda activate bm_hil
+$ conda env update -f environment.yml
+```
+
+Once you're done, you can always deactivate the environment with:
+
+```
+$ conda deactivate
+```
+
+Whenever working with HIL testing ensure to activate the environment with
+
+```
+$ conda activate bm_hil
+```
+
+## Running HIL Tests
+
+HIL tests require a serial communication to the device being tested.
+The requirements for this serial interface are as so:
+
+- Byte Size = 8 bits
+- Parity = None
+- Stop Bits = 1
+- Flow Control = False
+
+A specified baud rate is not a requirement.
+
+In order to run the tests the following command must be invoked:
+
+`pytest -s @tests_to_run.txt --port '{port}' --baud {baud}`
```

Comment:
I was really trying to get `pytest` with no args (other than port which is obviously required) to work, but it required too many file structure somersaults, explicit relative path additions, and pytest config files.... which is to say — great job! I don't think it can get any cleaner than this. 🙌